django_fsm/__init__.py

```diff
@@ -495,6 +495,32 @@ def save(self, *args, **kwargs):
         self._update_initial_state()
 
 
+class FSMModelMixin(object):
+    """
+    Mixin that allows refresh_from_db for models with fsm protected fields
+    """
+
+    def _get_protected_fsm_fields(self):
+        def is_fsm_and_protected(f):
+            return isinstance(f, FSMFieldMixin) and f.protected
+        protected_fields = filter(is_fsm_and_protected, self._meta.concrete_fields)
```

Comment:
Very nice, we have a few models that have multiple FSM fields in a single model. Happy to see that you are looking for more than just 1 field on a model.

Reply:
Thanks!, do you know how I can get this PR merged?